statsmodels/tsa/tests/test_arima.py

```diff
@@ -2092,7 +2092,10 @@ def test_arima111_predict_exog_2127():
     # rescaling results in convergence failure
     #model = sm.tsa.ARIMA(np.array(ef)*100, (1,1,1), exog=ue)
     model = ARIMA(ef, (1,1,1), exog=ue)
-    res = model.fit(transparams=False, iprint=0, disp=0)
+    res = model.fit(transparams=False, iprint=0, disp=0,
+                    maxiter=500)
+    import warnings
+    warnings.warn(res.params, SyntaxWarning)
```

Comment:
What is this SyntaxWarning?

Reply:
That is just to output parameters, temporarily.  This PR is failing now :-/